Fls 1414 header change

nuwan-samarasinghe opened the pull request:
_Add ticket reference to Pull Request title: e.g. 'FS-123: Add content', if there is no ticket prefix with BAU_


### Change description
_A brief description of the pull request_

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")


### How to test
_If manual testing is needed, give suggested testing steps_


### Screenshots of UI changes (if applicable)

israr-ulhaq reviewed (commented):
Showing GOV.UK if you open any form and on summary page as well.

<img width="1216" height="857" alt="Screenshot 2025-08-05 at 12 11 41" src="https://github.com/user-attachments/assets/bac07eca-9f18-459e-9175-995db73cf632" />

<img width="1222" height="827" alt="Screenshot 2025-08-05 at 12 13 48" src="https://github.com/user-attachments/assets/a0d443b0-51b4-42e4-900c-246d88b158ba" />

nuwan-samarasinghe commented:
@wjrm500 

Steps

1. Delete the .venv local
2. Delete all the docker images 
3. Run docker command `docker system prune`
4. Run `uv sync` for the project 
5. Run `make pre up` wait till all start 

if these steps don't work check the dev environment since it works and I don't have any other solutions since it works for @israr-ulhaq and myself

wjrm500 commented:
@nuwan-samarasinghe have followed all of those steps and still get the same error 😭

nuwan-samarasinghe commented:
Hi @wjrm500 sorry I forgot to update the comments I have fixed some of the cosmetic issues that you have mentioned and will try to fix some more today and I have updated the PR description with the latest screens

nuwan-samarasinghe commented:
Please check the new screen shots added to the PR description

wjrm500 reviewed (changes requested):
Hi @nuwan-samarasinghe thanks for this, the new header looks great - very neat, and consistent across the different services

I've left a few comments and also need to flag an important issue I found when testing locally: the accordion component in Assess is broken...

On **Test** environment - notice the "Show" action below each fund name:

<img width="1487" height="433" alt="Image" src="https://github.com/user-attachments/assets/ef3515c3-3066-432a-bb42-17545f1402cd" />

Locally - the "Show" action is absent, and in fact we can't open the accordion at all: 

<img width="2499" height="489" alt="Image" src="https://github.com/user-attachments/assets/da21f8e5-f91a-4d3a-9339-682dc8db82cb" />

This makes it difficult to review the rest of Assess. I've been through Apply and Authenticator though, comparing them to Test environment, and they both look good

As mentioned elsewhere, I won't give final approval until we've had Design sign-off here - thanks for reaching out on Slack to Nima, Favour, Jenny and Chanel. Let's see what they say
